brands/shop/convenience.json

```
@@ -595,6 +595,15 @@
       "shop": "convenience"
     }
   },
+  "shop/supermarket|Dia & Go": {
```

Thanks - make sure to change the identifier too!
```suggestion
  "shop/convenience|Dia & Go": {
```